include/klee/Solver/Solver.h

```diff
@@ -10,11 +10,11 @@
 #ifndef KLEE_SOLVER_H
 #define KLEE_SOLVER_H
 
-#include "klee/CommandLine.h" // FIXME: This is just for CoreSolverType
-#include "klee/Expr.h"
-
 #include <vector>
 
+#include "../Basic/CommandLine.h" // FIXME: This is just for CoreSolverType
```

Comment:
I've tried my best to remove all the relative includes already present in KLEE, unfortunately I missed some. Good catch!